src/chunk.h

```diff
@@ -43,7 +46,7 @@ class Chunk
   Chunk(wstring src, wstring dest, wstring cor, wstring wbl)
   : source(src), target(dest), coref(cor), wblank(wbl), isBlank(false), isJoiner(false), rule(-1)
   {}
-  Chunk(wstring dest, vector<Chunk*>& children, int r = -1)
+  Chunk(wstring dest, vector<Chunk*>& children, int r)
```

Comment:
Why was this changed?

Reply:
It gives an error to have a default value in the declaration. The default value is there in the cc file so nothing really changes.

Reply:
I don't see the initializers in the .cc

Reply:
Right, I moved it earlier, but forgot to add it again. Added